index.d.ts

```diff
@@ -209,8 +209,8 @@ declare module "minecraft-launcher-core" {
     protected printVersion(): void;
     protected createRootDirectory(): void;
     protected createGameDirectory(): void;
-    protected async extractPackage(): void;
-    protected async getModifyJson(): any;
+    protected extractPackage(): Promise<void>;
+    protected getModifyJson(): Promise<any>;
```

Comment:
Why delete the `async` keyword ?

Reply:
Because `async` is not ment to use in interface and typescript compiller thow an error. See screenshot above.

You can repeat it by yoursef: 

install `minecraft-launcher-core`,
then paste this code below to your `.ts`  file and try to compile it.
```ts
import {  Client  } from 'minecraft-launcher-core'

const launcher = new Client()

```

Here is stackowerflow answer, that explains it better than me: https://stackoverflow.com/a/55890828

Reply:
You are not gonna merge it I suppose?